ui_automation/suits/test_login.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+from playwright.sync_api import expect
+from ui_automation.pages.login_page import LoginPage
+from ui_automation.pages import locators
+
+@pytest.mark.usefixtures("page")
+class TestLogin:
+
+    def test_successful_login(self, page):
+        login_page = LoginPage(page)
+
+        login_page.open()
+        login_page.login("test.playwright@ukr.net", "q!RamZWyGBa4Z!j")
+        expect(page).to_have_url("https://mail.ukr.net/desktop*", timeout=10000)
```

Comment:
to_have_url matches either string or regex. So you need to pass full string or do something like this
```suggestion
        expect(page).to_have_url(re.compile(r"https://mail\.ukr\.net/desktop/.*"), timeout=10000)
```
and import re module of course